### ISSUE TITLE
[Layout v2] Caching too agressive

### ISSUE THREAD

**Issue by Syphdias** [firefly-iii/firefly-iii#5376]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

After viewing the accounts page once, transactions are no longer being updated. Added categories do not show up (possibly also modified, but I did not test that). Added transactions do not show up.

I suspect missing cache invalidation.

### Expected behaviour

Cache is invalidated and up-to-date data is shown.

### Debug information

Debug information generated at 2021-12-09 18:24:10 Europe/Berlin for Firefly III version **5.6.5**.

| Scope | Version |
| --- | --- |
| Firefly III | 5.6.5 |
| Firefly III API | 1.5.4 |
| PHP | 8.0.12 |
| Host | Linux |

| System info | Value |
| --- | --- |
| Installation ID | fd915216-470f-46b1-8459-a68fb47d6e4a |
| Using docker? | true |
| Telemetry | false |
| Layout | v2 |
| System TZ | Europe/Berlin |
| Browser TZ | Europe/Berlin |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | notice, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | apache2handler |
| Default language | en_US |
| Default locale | de_DE |
| BCscale | 24 |
| DB drivers | mysql, pgsql, sqlite |
| Current driver | mysql |
| DB version | 18 (exp. 18) |
| Login provider | eloquent  |
| Trusted proxies (.env) | ** |

| User info | Value |
| --- | --- |
| Session start | 2021-01-01 00:00:00 |
| Session end | 2021-12-31 23:59:59 |
| Session first | 2014-09-14 00:00:00 |
| User ID | 1 |
| User language | en_US |
| User locale | en_US |
| Attempt at "en_US.utf8" | 'en_US.utf8' |
| Attempt at "en_US.UTF-8" | 'en_US.UTF-8' |
| User agent | Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.93 Safari/537.36 |

### Steps to reproduce

1. Create transaction (Description "Test 1", Source "asset account 1",  Destination "asset account 2", amount 1) -> "Store transaction"
2. "Accounts" > "Asset accounts"
3. Pick "asset account 1"
4. "Action" > "Edit" the transaction and **give it a category** -> "Update transaction"
5. Create transaction (Description "Test 2", Source "asset account 1",  Destination "asset account 2", amount 2) -> "Store transaction"
6. "Accounts" > "Asset accounts"
7. Pick "asset account 1"
8. **Bug: Transaction "Test 1" has no category**
9. **Bug: Transaction "Test 2" does not show up**

**Comment by Syphdias** [comment]
Is there a workaround to clear the cache? <kbd>Ctrl-Shift-R</kbd> seems to be not enough.

**Comment by JC5** [comment]
Yes, the reload button on the account list should work! It's aggressive but that button should reset it.
